extensions/azurecore/src/extension.ts

```diff
@@ -82,23 +82,12 @@ export async function activate(context: vscode.ExtensionContext) {
 	};
 }
 
-async function exists(path: string): Promise<boolean> {
-	try {
-		await fs.access(path);
-		return true;
-	} catch (e) {
-		return false;
-	}
-}
-
 // Create the folder for storing the token caches
 async function findOrMakeStoragePath() {
 	let storagePath = path.join(getDefaultLogLocation(), constants.extensionName);
 	try {
-		if (!(await exists(storagePath))) {
-			await fs.mkdir(storagePath);
-			console.log('Initialized Azure account extension storage.');
-		}
+		await fs.mkdir(storagePath, { recursive: true });
```

Comment:
Not sure this is the right logic based on the console messages below. I image with this always throw if it exists (99% of the time) which then will show the messages below.

Reply:
I double checked for recursive mode it doesn't throw anything. Just silently ignores it as it already exists.